metadata.json

```diff
@@ -1,16 +1,17 @@
 {
   "name": "puppetlabs-ntp",
-  "version": "4.2.0",
+  "version": "6.0.0",
   "author": "Puppet Labs",
   "summary": "Installs, configures, and manages the NTP service.",
   "license": "Apache-2.0",
   "source": "https://github.com/puppetlabs/puppetlabs-ntp",
   "project_page": "https://github.com/puppetlabs/puppetlabs-ntp",
   "issues_url": "https://tickets.puppetlabs.com/browse/MODULES",
   "dependencies": [
-    {"name":"puppetlabs/stdlib","version_requirement":">= 4.6.0 < 5.0.0"}
+    { "name":"puppetlabs/stdlib","version_requirement":">= 4.13.0 < 5.0.0" },
+    { "name":"puppet/tea","version_requirement":">= 0" }
```

Comment:
hmm, as `puppet-tea` is a tiny, experimental module, and it's below 1.0 so no API guarantees, perhaps it'd be better to define the type aliases here inside ntp, and eliminate this dependency? alternately maybe we could pull those aliases over into stdlib or some place that they will get visibility and use but not restrict `puppet-tea`  from changing API? // cc @igalic